### ISSUE TITLE
WAILA Fixes

### ISSUE THREAD

**Issue by bartimaeusnek** [GTNewHorizons/NewHorizons#4012]
Since @mitchej123 wanted to fix the issues when i made a issue for them here they are:

- [ ] https://github.com/GTNewHorizons/NewHorizons/issues/2618
- [ ] https://github.com/squeek502/WailaHarvestability/issues/52
- [x] Gregtech Specific Sides (transformer "high"/ "low" side / Batterybuffer input/output side, etc)
- [ ] Malisis Door's are shown as "unknown Block - contact mod author"  
- [x] IC2 Crops do not show information after beeing scanned.

**Comment by draknyte1** [comment]
> Malisis Door's are shown as "unknown Block - contact mod author"

Is probably due to MD not registering a block properly during init.. YMMV fixing that from the other side if that's the case. 

Cables and wires definitely used to show the correct tool.. so I will check for you later if it's borked in master and try figure out when if so. If not, it's just an easy to find issue here (Hopefully just a .lang issue tbh)

**Comment by mitchej123** [comment]
![step_up](https://user-images.githubusercontent.com/1894689/51583624-5a3e6400-1e86-11e9-950f-762d0dd57470.png)

I've since updated it so that it'll say "Output: East" or "Input: East" depending on step up/step down.

**Comment by mitchej123** [comment]
Might also add ic2 crops support, especially after you've scanned the plant and know what it is.

**Comment by DoomSquirter** [comment]
https://imgur.com/a/lskDfJP

so mitch I don't have alot of higher tech stuff.  still no assline...  but I'm really not seeing a whole lot of things that aren't globally applicable.
I think all I've really seen are the following:
* problems variable or extra text for regular machines, to show there's an issue.  machines give more verbose of an error msg
* progress (in x/x seconds) for machines.
* voltage and amperage for cables/wires.  would be swell to not just throw 512 but actually say HV or something.
* batbuffs.  scanner only shows total voltage current/max eu.  I know that info panel shows batteries as well but scanner does not.  side note, info panel is buggy with batbuff so prob makes sense to just follow what scanner shows.  showing current/max eu for batbuffs would be better than nothing.

**Comment by mitchej123** [comment]
Mostly implemented; open a new issue if there's anything else to add.
